### ISSUE TITLE
Wrong line number for ANN201 with certain multiline docstrings

### ISSUE THREAD

**Issue by erikced** [sco1/flake8-annotations#106]
**Describe the bug**
ANN201 is reported for the wrong line when there is a multiline docstring only containing newline (or just a summary and a newline). Most likely related to  #32 and #77.

**To Reproduce**
Minimal code example to reproduce the behavior:

```py
def a():
    """
    """
    return [
        1,
        2,
        3
    ]
```

```
$ flake8 test.py
test.py:-6:1: ANN201 Missing return type annotation for public function
```

**Version Information**
Please provide the full output of `flake8 --version`

```bash
$ flake8 --version
3.8.4 (flake8-annotations: 2.5.0, flake8-bugbear: 20.11.1,
flake8-comprehensions: 3.3.1, flake8-eradicate: 1.0.0, mccabe: 0.6.1,
pycodestyle: 2.6.0, pyflakes: 2.2.0) CPython 3.7.9 on Linux
```

As well as your Python version:
```bash
$ python -V
Python 3.7.9
```

**Comment by sco1** [comment]
Thanks for the report! I'll try to take a look this weekend.

**Comment by sco1** [comment]
This should be fixed now on the dev branch; basically a sneaky off-by-one error.

I'll work on reviewing the remainder of #105 and try to get the release out in the next few days.

**Comment by erikced** [comment]
Great! Thank you for the quick response and resolution.
